DANS-KNAW/easy-module-archetype
# Run script rights

fixes an issue with the file rights of an application when deployed on (d | t)?easy

#### When applied it will
* make sure the run script is executable

#### Where should the reviewer @DANS-KNAW/easy start?
`src/main/resources/archetype-resources/src/main/assembly/bin.xml`

#### How should this be manually tested?
* create a new project based on this archetype and make it do something useless
* update dtap such that it can be deployed on (d | t)?easy
* deploy the new project on (d | t)?easy
* check that `ls /opt/easy-license-creator/bin/easy-license-creator -la` has `-rwxr-xr-x.` for its rights

or

* just verify that this change is consistent with all other projects 😄 

#### related pull requests on github
*none*
